browser/psst/brave_psst_utils_unittest.cc

```diff
@@ -0,0 +1,179 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/psst/browser/core/brave_psst_utils.h"
+
+#include "base/test/scoped_feature_list.h"
+#include "brave/components/psst/common/features.h"
+#include "brave/components/psst/common/psst_metadata_schema.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
+#include "chrome/test/base/testing_profile.h"
+#include "components/content_settings/core/browser/host_content_settings_map.h"
+#include "content/public/test/browser_task_environment.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace {
+
+base::Value::List VectorToList(const std::vector<std::string>& values) {
+  base::Value::List list;
+  for (auto& value : values) {
+    list.Append(value);
+  }
+  return list;
+}
+
+base::Value::Dict CreatePsstPermissionDict(
```

Comment:
this is not permission, this is PSST settings. Please remove any mention of "Permission" from this work to make this less confusing for everyone.